GettingStarted/ibmdbpy_GettingStarted_1-basics.ipynb

```diff
@@ -0,0 +1,2487 @@
+    "\n",
+    "To overwrite the `IRIS` table:\n",
+    "> idadf.save_as(tablename = \"IRIS\", clear_existing = True)\n",
+    "\n",
+    "To create a new table:\n",
+    "> idadf.save_as(tablename = \"NEW_NAME\", clear_existing = True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/eva.feillet/anaconda3/lib/python3.7/site-packages/ibmdbpy/frame.py:1899: UserWarning: Table TEST already exists.\n",
+      "  warnings.warn(message, UserWarning)\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Are you sure that you want to overwrite TEST [y/n] y\n"
+     ]
+    }
+   ],
+   "source": [
+    "idadf.save_as(tablename = \"TEST\", clear_existing = True)\n",
+    "# note: we have set clear_existing to True in case you run the notebook several times"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "__Wrap-up__\n",
+    "\n",
+    "Nice job! You are now familiar with operations on IdaDataFrames with ibmdbpy. Let's go to the next section, where you see how to use ibmdbpy's machine learning functionalities."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Apply Machine Learning techniques with ibmdbpy\n",
+    "\n",
+    "Not only does ibmdbpy provide you with in-database Pandas-like functionalities, but also with in-database Scikit-learn-like functionalities. Algorithms dedicated to KMeans clustering, association-rule mining and Naive Bayes classification are provided in `ibmdbpy.learn`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "__Example: KMeans algorithm__"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ibmdbpy.learn import KMeans\n",
+    "kmeans = KMeans(3) # configure clustering with 3 cluters"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note: In-DataBase Kmeans needs an indexer to identify each row. As checked above this is the case for our IdaDataFrame. We can go ahead with model fitting and prediction!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Uploading 150 rows (maxnrow was set to 20000)\n",
+      "ID\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Taking a fresh new start\n",
+    "from ibmdbpy.sampledata import iris\n",
+    "idadf = idadb.as_idadataframe(iris, \"IRIS\", clear_e
```

Comment:
Can you turn the entries here into links as well?

Reply:
I don't understand what you mean. Why would I use a link in this line of code?

Reply:
This is not a line of code. It's the filename of https://github.com/ibmdbanalytics/ibmdbpy-notebooks/blob/ibmdbpy-demos_eva-feillet/GettingStarted/ibmdbpy_GettingStarted_2-geo_basics.ipynb, so why not make it a link for users to click on?

Reply:
Ah, I see what you mean. For some reason, in the overview, this comment shows up on the wrong line. It's supposed to be at https://github.com/ibmdbanalytics/ibmdbpy-notebooks/pull/2/files#diff-2cd9705c73d36ce5f4b2e9cff90f7b3fR2440

```
    "* Getting started with ibmdbpy geospatial functions:\n",
    "        \n",
    "        ibmdbpy_GettingStarted_2-geo_basics\n",
```

Reply:
done